cmd/minikube/cmd/start.go

```diff
@@ -229,7 +242,7 @@ func clusterUp(config *cluster.MachineConfig) {
 	err = runner.Run(cmdName, cmdArgs...)
 	if err != nil {
 		// TODO glog is probably not right here. Need some sort of logging wrapper
-		glog.Errorln("Error starting 'cluster up': ", err)
+		glog.Errorln("Error starting the cluster: ", err)
```

Comment:
Well, I intentionally used 'cluster up', since this is the command which gets executed.